src/app/static/src/components/parameters/phasesUtils.ts

```diff
@@ -44,3 +44,20 @@ export const getDisplayPhase = (
 export const getTotalDays = (start: Date, end: Date): number => daysBetween(start, end) + 1;
 
 export const phaseClassFromIndex = (idx: number): string => (idx % 2 ? "phase-odd" : "phase-even");
+
+export const numericFormatter = (num: number): any => {
```

Comment:
This does look pretty solid to me, but could also have considered using [numeral.js](http://numeraljs.com/) as we have in HINT. I think we'll probably end up using that in dynamic form to format read only numbers.

If we do keep this util, I don't think this is right place for it, since it's not being used for anything to do with phases, or in any of the components. It could either go into a new utils file at top level src/app/static/src, or under store. 

It's a bit disorienting covering thousands, then billions, then millions, then sub-thousand, should probably either ascend or descend in magnitude. Could descend, and take advantage of the logic you've already used for millions, where you don't need to check the upper bound because you've already checked for billions:
```
if (num >= 1000000000) {...
if (num >= 1000000) {...
if (num >= 1000) {...
```